department-of-veterans-affairs/va.gov-cms
# VACMS-14205: Editors unable to save changes to Staff Profile pages

## Description

Closes #14205

## Testing done
Tested locally. All changes now save correctly. 

## Screenshots


## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As user _uid_ with _user_role_
1. Go to 'https://prod.cms.va.gov/va-jackson-health-care' and log in as log in as Camille.Ballentine@va.gov
2. Navigate to Staff Profiles ([Jason C. Cain i](https://staging.cms.va.gov/jackson-health-care/staff-profiles/jason-c-cain)s the prime example) 
3. Make a change, then Publish
   - [x] Validate that changes save. 
2. Then navigate to https://va-gov-cms.ddev.site/jackson-health-care/staff-profiles/james-burrell and edit the node, delete the biography fields and make sure the Create profile page with bio checkbox is unchecked and save
   - [x] Validate that it saves correctly and that the Bio fields are in fact removed. 

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
